ush/cdo_post_ocean_format.sh

```diff
@@ -0,0 +1,193 @@
+#         the `output_netcdf` file path.
+#
+#       * Assign the netCDF metadata attribute `coordinates` for
+#         variable `uo` the values `geolon_u,geolat_u,time` and update
+#         the `output_netcdf` file path.
+#
+#       * Assign the netCDF metadata attribute `coordinates` for
+#         variable `vo` the values `geolon_v,geolat_v,time` and update
+#         the `output_netcdf` file path.
+#
+#     input_netcdf: The netCDF-formatted file path containing the
+#                     variables defined in `variable_file`.
+#
+#     output_netcdf: A netCDF-formatted file path to contain the
+#                      specified variables remapped to the destination
+#                      grid projection.
+#######
+
+# Collect the command line arguments and check the validity.
+variable_file="${1}"
+input_path="${2}"
+output_path="${3}"
+
+#######
+
+if [[ "$#" -ne 3 ]]; then
+    echo "Usage: $0 <variable_file> <input_path> <output_path>"
+    exit 100
+fi
+
+#######
+
+# _comma_split_string - Split a comma-delimited string into an array.
+#
+# Description:
+#   This function takes a comma-delimited string as input and splits
+#     it into an array. Each element in the resulting array is
+#     obtained by splitting the input string at commas and then
+#     removing leading and trailing spaces.
+#
+# Parameters:
+#   $1 - The comma-delimited string to split.
+#
+# Global Variables:
+#   global_array - An array containing the split elements.
+#
+# Example usage:
+#   _comma_split_string "item1,item2 item3,item4"
+#   for element in "${global_array[@]}"; do
+#       echo "$element"
+#   done
+#
+# This example will split the input string into individual elements
+#   and print each element on a separate line.
+function _comma_split_string() {
+    local string="${1}"
+
+    local local_array=()
+    global_array=()
+    IFS="," read -ra items <<< "${string}"
+    for item in "${items[@]}"; do
+        local_array+=("${item} ")
+    done
+    for item in "${local_array[@]}"; do
+        IFS=" " read -ra items <<< "${item}"
+        for element in "${items[@]}"; do
+            global_array+=("${element} ")
+        done
+    done
+}
+
+#######
+
+# _strip_whitespace - Remove whitespace from a string.
+#
+# Description:
+#   This function takes an input string and removes all whitespace
+#   characters (spaces, tabs, and newline characters) to produce a
+#   cleaned output string.
+#
+# Parameters:
+#   $1 - The input string from which whitespace will be removed.
+#
+# Return:
+#   The cleaned string with no whitespace.
+#
+# Example usage:
+#   cleaned_string=$(_strip_whitespace "   This is a string   with spaces  ")
+#   echo "Cleaned string: \"$cleaned_string\""
+#
+# This example will remove all leading, trailing, and internal
+# whitespace from the input string and display the cleaned result.
+function _strip_whitespace(){
+    local in_string="${1}"
+
+    out_string=$(echo "${in_string}" | $(command -v sed) "s/ //g")
```

Comment:
```suggestion
    out_string=${in_string//[[:space:]]}
```

Reply:
I attempted this during development and it was not working as I needed it to.

Reply:
Hmmm, it worked when I tried a test case with spaces and tabs in a shell. Try again in case this is slightly different than you tried, then if still doesn't work, do this:
```suggestion
    out_string=$(sed "s/ //g" <<< "${in_string}")
```

Ironically, I can't get mine to work using _sed_. It leaves the internal whitespace:
```
walter@ubuntu:~$ cat test2.bash 
a=" b 		c   "
echo "-->${a//[[:space:]]}<--"
echo "-->$(sed 's/ //g' <<< "${a}")<--"
echo "-->$(echo "${a}" | sed "s/ //g")<--"


walter@ubuntu:~$ bash test2.bash 
-->bc<--
-->b		c<--
-->b		c<--
```